index.html

```diff
@@ -35,12 +35,144 @@ <h3>I am a Software Developer</h3>
+                <div class="padding-card">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a href="#" class="fullWidth-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                <div class="padding-card">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a href="#" class="fullWidth-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                <div class="padding-card">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a href="#" class="fullWidth-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                <div class="padding-card">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a href="#" class="fullWidth-btn">See Project</a>
+            </article>
+        </section>
+
+        <section class="about-me">
+            <article class="intro-about">
+                <h2>About me</h2>
+                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don't
+                    hestiate to contact me.</p>
+                    <a href="#">Get My Resume</a>
+            </article>
+
+            <article class="knowldege">
+                <section>
+                    <hr>
```

Comment:
Please fix the distance between this line and the background it doesn't looks exactly like the design 
Your design 
![image](https://user-images.githubusercontent.com/74744787/164721746-3b6ebd43-4e68-417e-a2ef-f6b10c8aef52.png)

Figma design 
![image](https://user-images.githubusercontent.com/74744787/164722072-5c33379a-e5a9-4948-bfa1-9150be98d4da.png)